help.txt

```diff
@@ -1,5 +1,5 @@
 MANAGE PLUGINS
-  asdf plugin-add <name> <git-url>     Add git repo as plugin
+  asdf plugin-add <name> [<git-url>]   Add a plugin
```

Comment:
I think the help message here should be a little more detailed. Perhaps something like `Add a plugin from the plugin repo OR add a Git repo as a plugin by specifying name and repo url`?